_docs/ops/managing-users.md

```diff
@@ -51,13 +51,16 @@ If the user requesting a reset has any apps, routes, or services in their sandbo
      > I've reset your one-time password. To regain cloud.gov access, log in to cloud.gov again. After entering your username/password combination, you should be prompted to set up a new one-time password with your authenticator app (for example, 1password, Microsoft Authenticator, or Authy).  Since this reset removed your roles on orgs and spaces, you may need to request additional access from your Space Managers and Org Managers again. If you had a sandbox space, that has been reset and is available to you again.
 
 ## Managing Admins
+
 Make sure you have a copy of the [cg-scripts repository](https://github.com/18F/cg-scripts) so you have access to several utility scripts.
 
 ### Creating Admins
+
 First, target and get a token for the main CloudFoundry UAA, and make the user a CloudFoundry admin using their GSA email address.
 
 ```sh
 cd /path/to/cg-scripts
+# on a jumpbox, run this instead of the next two commands: ./uaa/login.sh
```

Comment:
Any reason to not just remove the two `uaac` commands and reference the `./uaa/login.sh` command directly?

Reply:
This doc does mention running these commands locally and not just from a jumpbox. And it looks like `uaa/login.sh` only works on a jumpbox? Thoughts?

Reply:
Ah. Understood.